index.js

```diff
@@ -1410,8 +1410,14 @@ function Runtime() {
                     callback.call(this);
             },
             '- respondsToSelector:': function (sel) {
-                if (selectorAsString(sel) in methods)
+                const selName = selectorAsString(sel);
+                const canonSelNames = Object.keys(methods)
+                    .filter(m => /([+\-])\s(\S+)/.exec(m) !== null)
+                    .map(m => m.split(' ')[1]);
```

Comment:
How about moving this up so it's only computed once, e.g. as a `Set` that we call `has()` on here?

Reply:
Ah, makes sense! Will do.